### PR TITLE
default to staff and private for create

### PR DESCRIPTION
**Pull request by fiji-flo**

### CODE REVIEW

**Comment by fiji-flo** [comment]
I'm not sure if we should pull those changes into the null profile (not create).

File: python-modules/cis_profile/cis_profile/data/user_profile_null_create.json [hunk 1]
```diff
@@ -256,7 +256,7 @@
       "last_modified": "1970-01-01T00:00:00Z",
       "created": "1970-01-01T00:00:00Z",
       "verified" : true,
-      "display": null
+      "display": "staff"
```
Comment (gdestuynder) [inline python-modules/cis_profile/cis_profile/data/user_profile_null_create.json]: this should stay null for first/last name no?
or at least something non-staff - im guessing default is staff for staff and private for non-staff

**Comment by gdestuynder** [comment]
yes if it goes into create it has to also go into null profile

File: python-modules/cis_profile/cis_profile/data/user_profile_null_create.json (hunk 1 above)
Comment (fiji-flo) [reply python-modules/cis_profile/cis_profile/data/user_profile_null_create.json]: Yes. I think we settled on making this the skeleton for staff. I'm still in favor of dumping thins on the publishers :P

Do we wanna keep this for now or do you wanna put logic into the publishers? Or should I create different skeletons?

My main goal is to have phone book parity import for this week.

**Comment by fiji-flo** [comment]
Okay, I set everything that should be set depending on staff/non-staff to null. Everything that's not (usually) set by hris or ldap defaults to private (or the most restrictive setting).

@gdestuynder could you take a look.

File: python-modules/cis_profile/cis_profile/data/user_profile_null.json [hunk 2]
```diff
@@ -256,7 +256,7 @@
       "last_modified": "1970-01-01T00:00:00Z",
       "created": "1970-01-01T00:00:00Z",
       "verified" : true,
-      "display": null
+      "display": "staff"
```
Comment (gdestuynder) [inline python-modules/cis_profile/cis_profile/data/user_profile_null.json]:
```suggestion
      "display": null
```
this is last_name

File: python-modules/cis_profile/cis_profile/data/user_profile_null_create.json [hunk 3]
```diff
@@ -1171,7 +1171,7 @@
         "last_modified": "1970-01-01T00:00:00Z",
         "created": "1970-01-01T00:00:00Z",
         "verified" : true,
-        "display": "ndaed"
```
Comment (gdestuynder) [inline python-modules/cis_profile/cis_profile/data/user_profile_null_create.json]: note I think these were actually OK as ndaed

**Comment by gdestuynder** [comment]
last_name handling differs from first_name so i didnt approve

that said as i read it i wonder if we really need to do this? Since publishers would have to set their own defaults anyway.. 
I can add these to hris/ldap publishers (and auth0 when we write that)

**Comment by gdestuynder** [comment]
will add to publishers instead as discussed

**Pull request closed without merging**